### PR TITLE
Describe new function for string_fucntion_example

### PR DESCRIPTION
**Pull request by tricharu**
Please review it and let me know your feedback

### CODE REVIEW

File: string_function_examples.py [hunk 1]
```diff
@@ -8,3 +8,10 @@
 
 #title() return the string which contains of First character of string in Capital
 print(s.title())  # How are you
+
+
+# upper() return the string in upper case
```
Comment (ricky-aug) [inline string_function_examples.py]: use multi line comment which is our coding standard

File: string_function_examples.py [hunk 2]
```diff
@@ -8,3 +8,10 @@
...
+print(s.upper()) #HOW ARE YOU
+
+#isupper return true or false, whether the string is in upper case or not
+print(s.isupper()) #False
```
Comment (ricky-aug) [inline string_function_examples.py]: Don't give the o/p it is not good

**Review by ricky-aug — CHANGES_REQUESTED** [review]

File: string_function_examples.py [hunk 3]
```diff
@@ -8,3 +8,12 @@
 
 #title() return the string which contains of First character of string in Capital
 print(s.title())  # How are you
+
+
+'''
+upper() return the string in upper case
+'''
```
Comment (tricharu) [inline string_function_examples.py:15]: provide the multi line comment

File: string_function_examples.py [hunk 4]
```diff
@@ -8,3 +8,12 @@
...
+print(s.upper()) #HOW ARE YOU
+
+#isupper return true or false, whether the string is in upper case or not
+print(s.isupper()) #may be true or false depends on the string and return value of isupper()
```
Comment (tricharu) [inline string_function_examples.py:19]: Changed the comment regarding the true and false

**Review by ricky-aug — APPROVED** [review]

**Pull request merged**